diff --git a/index.js b/index.js
index 7506a90d782d92cb3b58b8b8be75392092ba712e..3a10582ba1e9404bbb5dc3822c0e1c3f19567978 100644
--- a/index.js
+++ b/index.js
@@ -134,8 +134,10 @@ const plugin = function (defaults = {}) {
 						sourcemap: 'linked',
 						banner: { js: 'globalThis.global = globalThis;' },
 						loader: {
-							'.wasm': 'copy'
-						}
+							'.wasm': 'copy',
+							'.ttf': 'copy'
+						},
+						assetNames: '[name]'
 					});
 
 					if (result.warnings.length > 0) {
