diff --git a/index.js b/index.js
index c2cfdbd800930094caa10a3df25f446e700c36d9..4cc2d2a31a18c65a8aa4b8c28a13c6fc9db8a3ea 100644
--- a/index.js
+++ b/index.js
@@ -128,7 +128,8 @@ const plugin = function (defaults = {}) {
 						sourcemap: 'linked',
 						banner: { js: 'globalThis.global = globalThis;' },
 						loader: {
-							'.wasm': 'copy'
+							'.wasm': 'copy',
+							'.ttf': 'copy'
 						}
 					});
 
