diff --git a/dist/index.edge.js b/dist/index.edge.js
index 089354ce4b62dde0fed3482e028ffade3d34b750..1680cbdf93a34183b5f11a655266fbbab2e1ed1d 100644
--- a/dist/index.edge.js
+++ b/dist/index.edge.js
@@ -18567,6 +18567,7 @@ var Resvg2 = class extends Resvg {
 // src/index.edge.ts
 import resvg_wasm from "./resvg.wasm?module";
 import yoga_wasm from "./yoga.wasm?module";
+import fallback_font from "./noto-sans-v27-latin-regular.ttf";
 
 // src/emoji/index.ts
 var U200D = String.fromCharCode(8205);
@@ -18980,7 +18981,7 @@ function assertValue(v, errorMessage) {
 // src/index.edge.ts
 var initializedResvg = initWasm(resvg_wasm);
 var initializedYoga = initYoga(yoga_wasm).then((yoga2) => Rl(yoga2));
-var fallbackFont = fetch(new URL("./noto-sans-v27-latin-regular.ttf", import.meta.url)).then((res) => res.arrayBuffer());
+var fallbackFont = fetch(new URL(fallback_font, import.meta.url)).then((res) => res.arrayBuffer());
 var ImageResponse = class extends Response {
   constructor(element, options = {}) {
     const result = new ReadableStream({
